docs/process.md

```diff
@@ -44,42 +44,27 @@ When:
 
 Requirements:
 
- * [emscripten-releases build CI](https://ci.chromium.org/p/emscripten-releases/g/main/console)
-   is green on all OSes for the desired hash (where the hash is the git hash in
-   the
-   [emscripten-releases](https://chromium.googlesource.com/emscripten-releases)
-   repo, which then specifies through
-   [DEPS](https://chromium.googlesource.com/emscripten-releases/+/refs/heads/master/DEPS)
-   exactly which revisions to use in all other repos).
+ * [emscripten-releases build CI][waterfall] is green on all OSes for the
+   desired hash (where the hash is the git hash in the
+   [emscripten-releases][releases_repo] repo, which then specifies through
+   [DEPS][DEPS] exactly which revisions to use in all other repos).
  * [GitHub CI](https://github.com/emscripten-core/emscripten/branches) is green
-   on the `master` branch.
+   on the `main` branch.
 
 How:
 
-1. Open a PR for the emsdk to update
-   [emscripten-releases-tags.txt](https://github.com/emscripten-core/emsdk/blob/main/emscripten-releases-tags.txt),
-   adding the version and the hash. Updating the "latest" tag there to the new
-   release is possible, but can also be deferred if you want to do more testing
-   before users fetching "latest" get this release.
-2. Run [update_bazel_workspace.sh](https://github.com/emscripten-core/emsdk/blob/main/scripts/update_bazel_workspace.sh).
-   This will update the bazel toolchain based on the new "latest" tag in emscripten-releases-tags.txt.
-3. Tag the emsdk repo as well, on the commit that does the update, after it
-   lands on master.
-4. Update
-   [emscripten-version.txt](https://github.com/emscripten-core/emscripten/blob/main/emscripten-version.txt)
-   in the emscripten repo. This is a delayed update, in that the tag will refer
-   to the actual release, but the update to emscripten-version.txt is a new
-   commit to emscripten that happens later.
-   * To minimize the difference, we should pick hashes for releases that are
-     very recent, and try to avoid anything else landing in between - can ask
-     on irc/chat for people to not land anything, or do this at a time of day
-     when that's unlikely, etc.
-   * There is no need to open a PR for this change, you can optionally just
-     commit it directly.
-5. Tag the emscripten repo on the emscripten commit on which
-   `emscripten-version.txt` was updated. (This could also be the commit from the
-   DEPS file as well, but this way is less confusing when just working on the
-   emscripten repo, and the difference should only be one commit anyhow.)
+1. Run [`./scripts/create_release.py`][create_release] in the emsdk repository.
+   This script will update [emscripten-releases-tags.txt][emscripten_releases_tags],
+   adding a new version based.  You can either specify the desired hash, or let
+   the script pick the current tot build.  The script will create a new git
```

Comment:
Does the script check that the tot build is finished and has been uploaded yet?

Reply:
I think it find the most recent build were all builds have been uploaded.  If that isn't what is wanted to allow the user to specify the revision manually (or at least that is the plan).

Reply:
Makes sense. I guess manually picking a revision will be the most common use case if we aren't going to be closing the tree for releases.